builds.json

```diff
@@ -2,6 +2,65 @@
   "builds" :
   [
     {
+      "version": "7.0",
+      "packages": [
+        {
+          "package": "org.ga4gh.models",
+          "python_package": "ga4gh",
+          "version": "3.0.0",
+          "dependencies": []
+        },
+        {
+          "package": "org.gel.models.metrics.avro",
+          "python_package": "metrics",
+          "version": "1.1.0",
+          "dependencies": []
```

Comment:
we are missing a dependency here to `participant` as follows:
```
"dependencies": [
             "org.gel.models.participant.avro"
           ]
```
which makes me wonder if all changes in the latest release are included in this branch @Antonior26 ?